qemu/tests/rng_hotplug.py

```diff
@@ -111,15 +126,23 @@ def stop_rngd(vm):
             hotplug_rng(vm, new_dev)
             dev_list.append(new_dev)
 
-        # Run test after hotplug
-        if test_after_hotplug and i == xrange(repeat_times)[-1]:
-            restart_rngd(vm)
-            error_context.context("Run %s after hotplug" % test_after_hotplug,
-                                  logging.info)
-            utils_test.run_virt_sub_test(test, params, env,
-                                         test_after_hotplug)
+        #run rng test after hotplug
+        check_and_run_subtest(rng_basic_test)
+
+        #run reboot/shutdown after hotplug
+        check_and_run_subtest(pm_test_after_plug)
```

Comment:
For repeat hotplug/unplug rng device, is it necessary to do pm test?
Seems it's not consistent with docstring.

Reply:
No need,the pm test are optional only in -one_rng  variant